exercises/L-functions-nested/exercise.js

```diff
@@ -1,5 +1,26 @@
-var mentor1 = "Daniel";
-var mentor2 = "Irina";
-var mentor3 = "Mimi";
-var mentor4 = "Rob";
-var mentor5 = "Yohannes";
+
+
+function add(name) {
```

Comment:
'add' is not a good variable name for this. what does it mean here to 'add'? -

maybe a better name is `greetMentor(name)`